angular/core/root_controller.coffee

```diff
@@ -26,7 +26,16 @@ angular.module('loomioApp').controller 'RootController', ($scope, $timeout, $loc
     IntercomService.boot()
     MessageChannelService.subscribe()
 
+  setTitle = (title) ->
+    document.querySelector('title').text = _.trunc(title, 300) + ' | Loomio'
```

Comment:
I've just thought that a better way to do this would be
```
%title {{ pageTitle }}
```
in application/index.html.haml,

then 
```
$scope.pageTitle = "#{_.trunc(title, 300)} | Loomio"
```

rather than relying on querySelector

Reply:
Also, there's a bit of discomfort around where our 'grab bag' goes at the moment... sometimes it's AppConfig, other times Session, and I'm not sure which is better (or which means which).

I'd be tempted here to do
```
$rootScope.$broadcast 'pageTitle', title
```
```
# navbar.coffee
$scope.$on 'pageTitle', (_, title) -> $scope.title = title
```

to avoid a global state if we don't need one